Gateway/Request/PaymentAlipayBuilder.php

```diff
@@ -0,0 +1,46 @@
+<?php
+namespace Omise\Payment\Gateway\Request;
```

Comment:
@jacekstan just idea, would it help reduce some repeated code if we have `Gateway/Request/OffsitePaymentBuilder` class and move these common codes to the class then lets these Alipay and Internet Banking to inherit from it?

Reply:
Also, just spotted some inconsistency here, 
`InternetbankingPaymentOffsiteBuilder` 
and 
`PaymentAlipayBuilder`

Guess it should be `AlipayPaymentOffsiteBuilder` or `PaymentInternetbankingBuilder`?

Reply:
not sure if 2 constants and 1 dependency injection in constructor is something worth to create class for, but you are right, it all can be done in 1 class, check commit please.

Reply:
@jacekstan ic. but you already changed lol (well, that might be good idea cuz more payment methods are coming)